BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,13 @@ $xmlhttprequest,third-party,denyallow=linkvertise.com|fastly.net|statically.io|s
 !
 ! NOTE: Specific rules
 !
+fplstatistics.com##[style="float: left; width: 140px; border-left: 0px; padding:0px"]
+fplstatistics.com##[style="float: right; width: 160px;"]
+fplstatistics.com##[style="text-align:center; margin-top: 10px"]
+@@||fplstatistics.com^$generichide
+@@||pagead2.googlesyndication.com^$third-party,domain=fplstatistics.com
+@@||fundingchoicesmessages.google.com/i/$third-party,script,domain=fplstatistics.com
+@@||googleads.g.doubleclick.net^$third-party,domain=fplstatistics.com
```

Comment:
Antiadblock rules - to corresponding file.
Could you please fix it also using scriptlet?
`third-party` in this case it is looks redundant.

Reply:
@Alex-302 i removed 3p modifier, but i'm not entirely sure about that these are anti adb rules